app/controllers/manage_consents_controller.rb

```diff
@@ -77,6 +77,10 @@ def handle_confirm
       @consent.recorded_at = Time.zone.now
       @consent.save!
 
+      if @consent.response_refused?
+        @triage.update! status: "do_not_vaccinate", user: current_user
+      end
```

Comment:
This can be done more cleanly, I will come back to it in this PR in a bit...

Reply:
What state is the `patient_session` in , and why doesn't it need to change?

Reply:
It goes into `consent_refused` but in that state we can still edit consent. It should go into `triaged_do_not_vaccinate` which requires a triage, so that's what this PR is adding.

Reply:
In the interest of speed I'm leaving this in, but note that I think this `handle_confirm` method could be refactored a bit to make it clearer.

Reply:
Ah, I see, it updates `triage` and then runs the state transitions below. 👍